Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/worker/DataDonationAnalyticsScheduler.kt

```diff
@@ -0,0 +1,59 @@
+package de.rki.coronawarnapp.datadonation.analytics.worker
+
+import androidx.work.ExistingPeriodicWorkPolicy
+import androidx.work.ExistingWorkPolicy
+import androidx.work.WorkManager
+import dagger.Reusable
+import javax.inject.Inject
+
+/**
+ * We schedule two workers to allow for randomness in the execution times,
```

Comment:
We just need randomness across the userbase (to be nice to the server), not check on the users device, couldn't we achieve this by setting a random initial delay for the periodic worker?